vocabularies/Common.xml

```diff
@@ -695,6 +698,9 @@ The minimum structure is
         <Property Name="CollectionPath" Type="Edm.String" Nullable="false">
           <Annotation Term="Core.Description" String="Resource path of an OData collection with possible values, relative to the document containing the value list mapping" />
         </Property>
+        <Property Name="DistinctValuesSupported" Type="Edm.Boolean" Nullable="true" DefaultValue="false"> 
```

Comment:
```suggestion
        <Property Name="DistinctValuesSupported" Type="Edm.Boolean" DefaultValue="false"> 
```

Reply:
Why should Nullable be false here?

Reply:
We have a default value of `false`, so we can omit the property in an annotation if distinct values are not supported.

I don't know what `DistinctValuesSupported:null` would mean - how would the UI react?

Reply:
Same behavior as in the "false" case. Should this be the reason to forbid the null value?

Reply:
Yes, we don't have tri-state logic of "yes", "no", "don't know".

Reply:
"don't know" would be "no", but I see that the Nullable false is more explicit.